client/src/components/forms/NewUserForm/NewUserForm.tsx

```diff
@@ -0,0 +1,64 @@
+import type { FormRule } from 'antd';
+import { PASSWORD_REQUIRED, PASSWORDS_DO_NOT_MATCH, USERNAME_REQUIRED } from '@/utils';
+import { FormButton, FormHOC, FormInput, FormSelect } from '@/components';
+import type { Option, VolunteerType } from '@/utils';
+
+interface NewUserValues {
+  username: string;
+  password: string;
+  confirmPassword: string;
+  userType: VolunteerType;
+}
+
+const options: Option[] = [
+  { value: 'Wolontariusz', label: 'Wolontariusz' },
```

Comment:
value is `collector`.
Also we have `collectorcoordinator` - `Koordynator wolontariuszy`